lib/services/apiAppManagement/lib/apiApp.js

```diff
@@ -0,0 +1,56 @@
+/**
+* Copyright (c) Microsoft.  All rights reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+var common = require('azure-common');
+var ApiAppManagementClient = require('./apiAppManagementClient');
+var getDeploymentTemplate = require('./getDeploymentTemplate');
+
+exports.ApiAppManagementClient = ApiAppManagementClient.ApiAppManagementClient;
+
+//
+// New up a dummy client so we can get at the TemplateOperations
+// prototype and attach our custom method to it.
+//
+var dummyClient = new exports.ApiAppManagementClient(
+  new common.TokenCloudCredentials({subscriptionId: '0', token: '0'}),
+    'https://dummy.example');
+getDeploymentTemplate.attachGetDeploymentTemplate(dummyClient);
+
+/**
+* Create a new {@link ApiAppManagementClient} object.
+*
+* @param {object} credentials The credentials object (typically, a TokenCloudCredentials instance).
+* @param {array}  [filters]                         Optional array of service filters
```

Comment:
nit, should we put the "filters" behind the "baseUri", so the order will be consistent with the function signature.